test/classes/initializers/generics/phase1/param-from-new.chpl

```diff
@@ -0,0 +1,20 @@
+
+record R {
+  var x : int;
+}
+
+class C {
+  param t;
```

Comment:
What happens if we do the same thing but via omitted initialization for the param field? E.g.
```chapel
class C {
  param t = new R();
  var v = new R();

  proc init() {
    v = new R();
    super.init();
  }
}
```

Reply:
That case is also caught by this PR.

Reply:
Excellent!  I figured it would be.  Your call whether to add a test for it or not